__tests__/app.test.js

```diff
@@ -290,6 +290,22 @@ describe("api/articles/:article_id/comments", () => {
     });
 });
 
+describe("/api/users", () => {
+    test("GET:200 sends an array of user objects with username, name and avatar_url properties", () => {
+        return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({ body }) => {
+                const { users } = body;
+                users.forEach((user) => {
```

Comment:
we need an assertion to check the length of the amount of users coming back before the forEach